requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask==0.10.1
+lxml==3.4.2
```

Comment:
The `requirements.txt` must include production dependencies. So, it misses `rpi.gpio==0.5.11`. Then we should have a `mock-requirements.txt` for web development. Already referenced [here](https://github.com/RuggedPOD/ruggedpod-web/blob/master/Vagrantfile#L20)